README.md

```diff
@@ -9,31 +9,17 @@ Being part of the Fediverse allows you to be free from data-harvesting corporati
 
 [Join Friendica]((https://dir.friendica.social/servers)) today or setup [your own Friendica instance](doc/Install.md).
 
-### Screenshots
-
-| ![Frio theme in mobile browser](images/screenshots/friendica-frio-mobile-profile-1.png?raw=true "Frio theme in mobile browser") ![Frio theme in mobile browser](images/screenshots/friendica-frio-mobile-profile-2.png?raw=true "Frio theme in mobile browser") |
```

Comment:
What about the old screenshots? Shouldn't we remove them?

Reply:
I figured these changes may not be to everyone's liking, so I kept the screenshots just in case. I know Git can help retrieve deleted files but you would need to know they existed in the first place.